cost/google/old_snapshots/google_delete_old_snapshots.pt

```diff
@@ -172,14 +172,14 @@ policy "policy_old_snapshots" do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} rows containing Old Snapshots"
     detail_template <<-EOS
 # Google Old Snapshots
-| Project ID | Name | Size | Labels | Age In Days| Status |
+| Project ID | Name | Size | Labels | Age In Days(Rounded)| Status |
```

Comment:
We don't need Rounded in the title.  Customers expect to see whole days.

Reply:
made the change